pkg/aws/ec2/api/wrapper.go

```diff
@@ -149,6 +149,13 @@ var (
 		},
 	)
 
+	numAssignedIPv4Prefixes = prometheus.NewCounter(
```

Comment:
nit: maybe you want to be consistent to either use allocated or assigned. No need to change in this PR. If you want to do, let's make the change in the next PR.